src/persist-client/src/iter.rs

```diff
@@ -678,16 +688,36 @@ where
     /// Wait until data is available, then return an iterator over the next
     /// consolidated chunk of output. If this method returns `None`, that all the data has been
     /// exhausted and the full consolidated dataset has been returned.
-    #[allow(dead_code)]
```

Comment:
Was this previously `dead_code` or just a left over annotation?

Reply:
Left over - it's cleared out in the initial "cleanup" commit.